pkgdb2/ui/packages.py

```diff
@@ -1130,14 +1130,24 @@ def package_request_branch(package, full=True):
 def package_request_new():
     ''' Page to request a new package. '''
 
-    collections = pkgdb2.lib.search_collection(
-        SESSION, '*', 'Under Development')
-    collections.extend(pkgdb2.lib.search_collection(SESSION, '*', 'Active'))
-    pkg_status = pkgdb2.lib.get_status(SESSION, 'pkg_status')['pkg_status']
+    collections = pkgdb2.lib.search_collection(SESSION, '*', 'Under Development')
+    collections.reverse()
+    active_collections = pkgdb2.lib.search_collection(SESSION, '*', 'Active')
+    active_collections.reverse()
+    # We want all the branch `Under Development` as well as all the `Active`
```

Comment:
It would be much better to be able to retrieve the list of active branches from sort of releng web service... but that doesn't exist.  ;)  This will do.